lms/djangoapps/static_template_view/views.py

```diff
@@ -50,7 +52,16 @@ def render(request, template):
         # This is necessary for the dialog presented with the TOS in /register
         if template == 'honor.html':
             context['allow_iframing'] = True
-        return render_to_response('static_templates/' + template, context, content_type=content_type)
+        # Format Examples: static_template_about_header
+        configuration_base = 'static_template_' + template.replace('.html', '').replace('-', '_')
+        page_header = configuration_helpers.get_value(configuration_base + '_header')
+        page_content = configuration_helpers.get_value(configuration_base + '_content')
+        if page_header:
+            context['page_header'] = SafeUnicode(page_header)
+        if page_content:
+            context['page_content'] = SafeUnicode(page_content)
```

Comment:
I'm a little confused why this is the first use of `SafeUnicode` or `SafeText` in edx-platform.  How has this problem been solved in other places?

Reply:
`mark_safe` is used a few times in edx-platform, and will accomplish the same thing (but more flexibly).  I'm still a little confused why something like this doesn't appear more often.

Reply:
I noticed the same thing as I am looking through it. I chose SafeUnicode because its the standard Django approach. As far as why its not entirely sure why there isn't more. I'm switching it over now.